src/main/java/com/dzionek/simpleshell/user/UserInput.java

```diff
@@ -0,0 +1,18 @@
+package com.dzionek.simpleshell.user;
+
+/**
+ * @author Bartlomiej Janik
+ * @since 9/16/2017
+ */
+public class UserInput {
+
+    private String[] userInput;
+
+    public void setUserInput(String userInput) {
+        this.userInput = userInput.split(" ");
```

Comment:
split by regex : [\s] to divide by every whitespace
https://regexone.com/lesson/whitespaces